tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -67,6 +74,20 @@ func (mcs *MultiClientSnapshotter) ConnectOrCreateRepo(repoPath string) error {
 	return err
 }
 
+// SetCacheSizeLimits sets the cache size limits for an existing repository
+// the repository server is connected to.
+// Allowed flags: content-cache-size-limit-mb, metadata-cache-size-limit-mb.
+func (mcs *MultiClientSnapshotter) SetCacheSizeLimits(cacheFlag, cacheFlagValue string) error {
+	if !strings.Contains(cacheFlag, contentCacheLimitMBFlag) && !strings.Contains(cacheFlag, metadataCacheLimitMBFlag) {
+		return ErrUnsupportedFlagsForCacheSet
+	}
+
+	_, _, err := mcs.server.Run("cache", "set",
+		cacheFlag, cacheFlagValue)
```

Comment:
Convert `cacheSize` using `strconv.Itoa`

```suggestion
		cacheFlag, strconv.Itoa(cacheSize))
```